package.json

```diff
@@ -1,19 +1,22 @@
 {
   "name": "movies-db",
   "version": "1.0.0",
-  "description": "react-redux movies database",
+  "description": "react movie database",
   "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server",
-    "build:dev": "webpack --env dev",
-    "build:prod": "webpack --env prod",
+    "build:dev": "webpack-dev-server",
+    "build:prod": "webpack --mode=production",
```

Comment:
Is it ok to leave this way for development version?

Reply:
`npm start` is a common name for building development version and serving it on localhost, so I suggest to leave 'start' and 'build' options, the first one is webpack-dev-server, the latter is for "webpack --mode=production"